README.md

```diff
@@ -75,6 +75,13 @@ You can add additional meta files in this directory to cover your own project fi
 [PhpStorm advanced metadata](https://www.jetbrains.com/help/phpstorm/ide-advanced-metadata.html)
 for more information.
 
+## Development Environment with ddev
+- Install ddev
+- Clone the repository as described in Installation -> Using Git
+- Create a ddev config using ```$ ddev config``` the defaults should be good for you
+- Open .ddev/config.yaml and change the php version to 7.2
+- Navigate to http://magento-lts.ddev.site
```

Comment:
With DDEV you could navigate to https://...

Reply:
@kkrieger85 It isn't set up properly in my env. yet. Something went wrong with the mkcert.

Reply:
Solution fix it and change the link to https?

Reply:
Should be resolved in the newest commit.